SeaPublicWebsite/Views/EnergyEfficiency/CavityWallsInsulated.cshtml

```diff
@@ -27,17 +27,59 @@
             @(Html.HiddenFor(m => m.EntryPoint))
             @Html.AntiForgeryToken()
             
+            @{
+                Func<object, object> details = 
+                    @<text>
+                       @if (Model.YearBuilt is >= 1991)
+                       {
+                           <p class="govuk-body">
+                               Homes built after 1991 were typically built with insulating material between the outer and inner walls. 
+                           </p>
+                           <p class="govuk-body">
+                               If your home was converted, for example it was a shop or other commercial premises before, 
+                               it may have been built without insulation.
+                           </p>
+                       }
+                       else
+                       {
+                           <p class="govuk-body">
+                               Many cavity walls can be insulated by injecting insulation material into the cavity from the outside. 
+                               A specialist company will drill holes in the outside walls, inject insulation through the holes and 
+                               then seal them with cement.
+                           </p>
+                           <p class="govuk-body">
+                               If there are small, filled 22 millimetre holes at regular intervals around the outside walls, 
+                               it’s a sign they may have been insulated.
+                           </p>
+                       }
+                     </text>;
+                
+                Func<object, object> epcHint = null;
+                if (Model.Epc?.CavityWallsInsulated is not null)
+                {
+                    epcHint = 
+                        @<p class="govuk-body">
+                            Your @Model.Epc?.LodgementDate Energy Performance Certificate suggests you have
```

Comment:
I think LodgementDate here is supposed to just be the year, will this show the exact date?

Reply:
You can add a readonly computed property to the EPC class to just get the year if necessary
The LodgementDate in the EpcDto is a string, should it be a DateTime?